js/modules/k6/html/html.go

```diff
@@ -52,6 +53,11 @@ type Selection struct {
 	URL string
 }
 
+type FormValue struct {
+	name  string
```

Comment:
These should be exported. They should also probably be in a separate file; `serialize.go` or something of the sort.

The sheer amount of functionality in this module means we have to be careful when adding stuff to the main file, lest it ends up even more unreadable.

Reply:
What do you mean by 'these should be exported'. Note that I'm only a Go programmer since 2 days ;).

Reply:
I found it. Kind of weird that you have to export stuff in Go by using uppercase.